src/main/groovy/io/cify/runner/utils/CapabilityParser.groovy

```diff
@@ -169,13 +253,13 @@ class CapabilityParser {
      * Gets external parameters for desired capabilities
      *
      * @return map of capabilities
-     * */
-    static LazyMap parseExtraCapabilities(String extraCapabilities) {
+     */
+    private static LazyMap parseExtraCapabilities(String extraCapabilities) {
 
         LazyMap result
 
         if (extraCapabilities.isEmpty()) {
-            result = [:]
+            result = new LazyMap()
         } else {
```

Comment:
Could you please add a short comment what is happening here: "extraCapabilities.split('&').inject([:]) { map, token ->
                token.split('=').with { map[it[0]] = it[1] }	                token.split('=').with { map[it[0]] = it[1]" ?

Reply:
This was a part of extra capabilities parsing. This functionality is dropped